lib/backend/k8s/k8s_fv_test.go

```diff
@@ -516,140 +491,143 @@ var _ = Describe("Test Syncer API for Kubernetes backend", func() {
 		})
 	}()
 
-	It("should handle a CRUD of System Network Policy", func() {
-		// In the backend, the Policy name is prepended to indicate where
-		// the policy is derived from in KDD.  The System Network Policy
-		// is a TPR and in the backend the name is prepended with
-		// "snp.projectcalico.org/".  The SNP CRUD operations assume that
-		// the name is of the correct format (it's up to the calling code
-		// to fan-out Policy CRUD operations to the appropriate KDD client
-		// based on the prefix).
-		kvp1Name := "snp.projectcalico.org/my-test-snp"
+	It("should handle a CRUD of Global Network Policy", func() {
+
+		kvp1Name := "my-test-gnp"
 		kvp1a := &model.KVPair{
 			Key:   model.PolicyKey{Name: kvp1Name},
 			Value: &calicoAllowPolicyModel,
 		}
-		kvp1b := &model.KVPair{
```

Comment:
> or an Update operation which needs a revision number 

I don't think that's true, now that Update does a Get first.  Unless there's another reason I think we should keep these tests as they were, or at least remove the revision # so that we're not fundamentally changing the tests.

We can add another test that sets the revision on Update.